fs/ggml/ggml.go

```diff
@@ -104,6 +107,28 @@ func (kv KV) Bool(key string, defaultValue ...bool) bool {
 	return keyValue(kv, key, append(defaultValue, false)...)
 }
 
+func (kv KV) UintOrMaxArrayValue(key string, defaultValue ...uint32) uint32 {
```

Comment:
```suggestion
func (kv KV) UintOrMaxArrayValue(key string, defaultValue uint32) uint32 {
```

Do we need a vararg here?

Reply:
good point, I was copying the same pattern used elsewhere in this file for providing default defaults (😁), but since I don't actually make use it in any of these, I've simplified it in the next commit